[WARN ][2014-04-30 13:03:00,877][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:28:17,581][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:28:18,678][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:28:18,874][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:28:41,574][com.mchange.v2.resourcepool.BasicResourcePool] - com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@6057f993 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (1). Last acquisition attempt exception: 
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2043][11550][3.64.96] 异常 java.net.ConnectException：打开端口 38,500 上服务器 office2.cherrypicks.com/115.160.176.82 的套接字时出错，消息为：Connection timed out: connect。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.am.bd.a(bd.java:340)
	at com.ibm.db2.jcc.t4.vb.a(vb.java:433)
	at com.ibm.db2.jcc.t4.vb.<init>(vb.java:93)
	at com.ibm.db2.jcc.t4.a.b(a.java:356)
	at com.ibm.db2.jcc.t4.b.newAgent_(b.java:2028)
	at com.ibm.db2.jcc.am.Connection.initConnection(Connection.java:720)
	at com.ibm.db2.jcc.am.Connection.<init>(Connection.java:669)
	at com.ibm.db2.jcc.t4.b.<init>(b.java:332)
	at com.ibm.db2.jcc.DB2SimpleDataSource.getConnection(DB2SimpleDataSource.java:232)
	at com.ibm.db2.jcc.DB2SimpleDataSource.getConnection(DB2SimpleDataSource.java:198)
	at com.ibm.db2.jcc.DB2Driver.connect(DB2Driver.java:475)
	at com.ibm.db2.jcc.DB2Driver.connect(DB2Driver.java:116)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:134)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:182)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:171)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:137)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1014)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$800(BasicResourcePool.java:32)
	at com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask.run(BasicResourcePool.java:1810)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:351)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:213)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:200)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at com.ibm.db2.jcc.t4.v.run(v.java:49)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.db2.jcc.t4.vb.a(vb.java:419)
	... 18 more
[WARN ][2014-04-30 13:29:03,031][com.mchange.v2.resourcepool.BasicResourcePool] - com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@7b5a5d5d -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (1). Last acquisition attempt exception: 
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2043][11550][3.64.96] 异常 java.net.ConnectException：打开端口 38,500 上服务器 office2.cherrypicks.com/115.160.176.82 的套接字时出错，消息为：Connection timed out: connect。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.am.bd.a(bd.java:340)
	at com.ibm.db2.jcc.t4.vb.a(vb.java:433)
	at com.ibm.db2.jcc.t4.vb.<init>(vb.java:93)
	at com.ibm.db2.jcc.t4.a.b(a.java:356)
	at com.ibm.db2.jcc.t4.b.newAgent_(b.java:2028)
	at com.ibm.db2.jcc.am.Connection.initConnection(Connection.java:720)
	at com.ibm.db2.jcc.am.Connection.<init>(Connection.java:669)
	at com.ibm.db2.jcc.t4.b.<init>(b.java:332)
	at com.ibm.db2.jcc.DB2SimpleDataSource.getConnection(DB2SimpleDataSource.java:232)
	at com.ibm.db2.jcc.DB2SimpleDataSource.getConnection(DB2SimpleDataSource.java:198)
	at com.ibm.db2.jcc.DB2Driver.connect(DB2Driver.java:475)
	at com.ibm.db2.jcc.DB2Driver.connect(DB2Driver.java:116)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:134)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:182)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:171)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:137)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1014)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$800(BasicResourcePool.java:32)
	at com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask.run(BasicResourcePool.java:1810)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:351)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:213)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:200)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:529)
	at com.ibm.db2.jcc.t4.v.run(v.java:49)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.db2.jcc.t4.vb.a(vb.java:419)
	... 18 more
[WARN ][2014-04-30 13:36:15,272][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:58:41,514][com.mchange.v2.async.ThreadPoolAsynchronousRunner] - com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@23905e3 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks!
[WARN ][2014-04-30 13:58:41,515][com.mchange.v2.async.ThreadPoolAsynchronousRunner] - com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@23905e3 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@482c94f (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#0)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@266e63d9 (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#1)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@195d4f45 (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#2)
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@3cff4805
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1bd15f8c
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@4ed7ee98
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@42716598
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@22c5003c
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@339274f9
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@543da1e2
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1a4f27a3
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@42b34bd3
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@9372bdb
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@5203b97
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@7283922
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@61beead9
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@3d2c7eb1
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@3431b5d7
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@2b995106
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@5e967746
Pool thread stack traces:
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(SocketInputStream.java:129)
		com.ibm.db2.jcc.t4.x.b(x.java:208)
		com.ibm.db2.jcc.t4.x.c(x.java:360)
		com.ibm.db2.jcc.t4.x.c(x.java:473)
		com.ibm.db2.jcc.t4.x.v(x.java:1241)
		com.ibm.db2.jcc.t4.ab.f(ab.java:97)
		com.ibm.db2.jcc.t4.p.e(p.java:81)
		com.ibm.db2.jcc.t4.qb.k(qb.java:160)
		com.ibm.db2.jcc.am.io.jb(io.java:2322)
		com.ibm.db2.jcc.am.jo.b(jo.java:4401)
		com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
		com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
		com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#1,5,main]
		java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
		java.net.InetAddress$1.lookupAllHostAddr(InetAddress.java:876)
		java.net.InetAddress.getAddressFromNameService(InetAddress.java:1229)
		java.net.InetAddress.getAllByName0(InetAddress.java:1180)
		java.net.InetAddress.getAllByName(InetAddress.java:1110)
		java.net.InetAddress.getAllByName(InetAddress.java:1046)
		java.net.InetAddress.getByName(InetAddress.java:996)
		com.ibm.db2.jcc.t4.a.a(a.java:1007)
		com.ibm.db2.jcc.t4.x.b(x.java:250)
		com.ibm.db2.jcc.t4.x.c(x.java:360)
		com.ibm.db2.jcc.t4.x.c(x.java:473)
		com.ibm.db2.jcc.t4.x.v(x.java:1241)
		com.ibm.db2.jcc.t4.ab.f(ab.java:97)
		com.ibm.db2.jcc.t4.p.e(p.java:81)
		com.ibm.db2.jcc.t4.qb.k(qb.java:160)
		com.ibm.db2.jcc.am.io.jb(io.java:2322)
		com.ibm.db2.jcc.am.jo.b(jo.java:4401)
		com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
		com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
		com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#2,5,main]
		java.lang.Object.wait(Native Method)
		java.lang.Object.wait(Object.java:485)
		java.net.InetAddress.checkLookupTable(InetAddress.java:1300)
		java.net.InetAddress.getAddressFromNameService(InetAddress.java:1217)
		java.net.InetAddress.getAllByName0(InetAddress.java:1180)
		java.net.InetAddress.getAllByName(InetAddress.java:1110)
		java.net.InetAddress.getAllByName(InetAddress.java:1046)
		java.net.InetAddress.getByName(InetAddress.java:996)
		com.ibm.db2.jcc.t4.a.a(a.java:1007)
		com.ibm.db2.jcc.t4.x.b(x.java:250)
		com.ibm.db2.jcc.t4.x.c(x.java:360)
		com.ibm.db2.jcc.t4.x.c(x.java:473)
		com.ibm.db2.jcc.t4.x.v(x.java:1241)
		com.ibm.db2.jcc.t4.ab.f(ab.java:97)
		com.ibm.db2.jcc.t4.p.e(p.java:81)
		com.ibm.db2.jcc.t4.qb.k(qb.java:160)
		com.ibm.db2.jcc.am.io.jb(io.java:2322)
		com.ibm.db2.jcc.am.jo.b(jo.java:4401)
		com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
		com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
		com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
		com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)


[WARN ][2014-04-30 13:58:41,889][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:58:41,889][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,637][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,637][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,644][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,645][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,645][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:00,645][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:01,696][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:05,482][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:11,514][com.mchange.v2.async.ThreadPoolAsynchronousRunner] - com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@23905e3 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks!
[WARN ][2014-04-30 13:59:11,515][com.mchange.v2.async.ThreadPoolAsynchronousRunner] - com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@23905e3 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@29557211 (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#1)
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@28b3cf4a (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#0)
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@34ee1826 (com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#2)
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@7b7381ef
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@5c57b13a
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@1348633
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@1f567ee3
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@67a47927
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@5f0e2499
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@1984e990
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@6501fa06
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@64c5d3b8
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@77cf08b7
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@394563d4
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@7a9d10ab
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@5db18970
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@32b6481e
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@5d38c394
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@39b0595c
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@68bac49a
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@3dc1902d
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@6ee09a07
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@5ba07a6f
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@4c63c68
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@72a7d24a
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@67df3314
		com.mchange.v2.resourcepool.BasicResourcePool$AcquireTask@6e929b52
Pool thread stack traces:
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#1,5,main]
		com.ibm.db2.jcc.am.Connection.close(Connection.java:1347)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:549)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:234)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.destroyResource(C3P0PooledConnectionPool.java:470)
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask.run(BasicResourcePool.java:964)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#0,5,main]
		com.ibm.db2.jcc.am.Connection.close(Connection.java:1347)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:549)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:234)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.destroyResource(C3P0PooledConnectionPool.java:470)
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask.run(BasicResourcePool.java:964)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)
	Thread[com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread-#2,5,main]
		com.ibm.db2.jcc.am.Connection.close(Connection.java:1347)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:549)
		com.mchange.v2.c3p0.impl.NewPooledConnection.close(NewPooledConnection.java:234)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.destroyResource(C3P0PooledConnectionPool.java:470)
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask.run(BasicResourcePool.java:964)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:547)


[WARN ][2014-04-30 13:59:19,765][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:41,516][com.mchange.v2.async.ThreadPoolAsynchronousRunner] - Task com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@482c94f (in deadlocked PoolThread) failed to complete in maximum time 60000ms. Trying interrupt().
[WARN ][2014-04-30 13:59:50,834][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
[WARN ][2014-04-30 13:59:50,844][com.mchange.v2.c3p0.impl.DefaultConnectionTester] - SQL State '08001' of Exception which occurred during a Connection test (fallback DatabaseMetaData test) implies that the database is invalid, and the pool should refill itself with fresh Connections.
com.ibm.db2.jcc.am.DisconnectNonTransientConnectionException: [jcc][t4][2030][11211][3.64.96] 在对连接的底层 socket、socket input stream 或 socket output stream 执行操作期间
发生通信错误。错误位置：Reply.fill() - socketInputStream.read (-1)。消息：Read timed out。 ERRORCODE=-4499, SQLSTATE=08001
	at com.ibm.db2.jcc.am.bd.a(bd.java:321)
	at com.ibm.db2.jcc.t4.a.a(a.java:548)
	at com.ibm.db2.jcc.t4.a.a(a.java:543)
	at com.ibm.db2.jcc.t4.x.b(x.java:252)
	at com.ibm.db2.jcc.t4.x.c(x.java:360)
	at com.ibm.db2.jcc.t4.x.c(x.java:473)
	at com.ibm.db2.jcc.t4.x.v(x.java:1241)
	at com.ibm.db2.jcc.t4.ab.f(ab.java:97)
	at com.ibm.db2.jcc.t4.p.e(p.java:81)
	at com.ibm.db2.jcc.t4.qb.k(qb.java:160)
	at com.ibm.db2.jcc.am.io.jb(io.java:2322)
	at com.ibm.db2.jcc.am.jo.b(jo.java:4401)
	at com.ibm.db2.jcc.am.CallableStatement.ec(CallableStatement.java:148)
	at com.ibm.db2.jcc.am.DatabaseMetaData.executeCatalogQuery(DatabaseMetaData.java:7813)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTablesX(DatabaseMetaData.java:6325)
	at com.ibm.db2.jcc.am.DatabaseMetaData.getTables(DatabaseMetaData.java:6231)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnectionNoQuery(DefaultConnectionTester.java:185)
	at com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:62)
	at com.mchange.v2.c3p0.AbstractConnectionTester.activeCheckConnection(AbstractConnectionTester.java:67)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:368)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:310)
	at com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:1999)
	at com.mchange.v2.async.ThreadPerTaskAsynchronousRunner$TaskThread.run(ThreadPerTaskAsynchronousRunner.java:255)
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.read(SocketInputStream.java:129)
	at com.ibm.db2.jcc.t4.x.b(x.java:208)
	... 19 more
